bin/extract_local_language_names.js

```diff
@@ -0,0 +1,25 @@
+const { readdirSync, readFileSync, writeFileSync } = require('fs')
+const { join } = require('path')
+
+const localeDir = join(__dirname, '..', '_locales')
+
+// English Lang as fallback
+const englishLang = JSON.parse(readFileSync(join(localeDir, 'en.json')))
```

Comment:
Note that you can just as well do

```js
const englishLang = require(join(localeDir, 'en.json'))
```

Reply:
We should prefer to use require as webpack has good support for it. Sure on a require you could somehow inline js code which gets executed but this shouldn't be a problem.

Reply:
I didn't thought of that, thanks for pointing it out to me.